packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.ts

```diff
@@ -131,7 +131,7 @@ export default baseMixins.extend({
         'v-navigation-drawer--floating': this.floating,
         'v-navigation-drawer--is-mobile': this.isMobile,
         'v-navigation-drawer--is-mouseover': this.isMouseover,
-        'v-navigation-drawer--mini-variant': this.miniVariant || (this.expandOnHover && !this.isMouseover),
+        'v-navigation-drawer--mini-variant': this.isMiniVariant || (this.expandOnHover && !this.isMouseover),
```

Comment:
looks like the second conditional can go into **isMiniVariant** as well since the only places that **miniVariant** was used also checks **expandOnHover** and **isMouseOver**.

Reply:
```suggestion
        'v-navigation-drawer--mini-variant': this.isMiniVariant,
```